compute/admin_guide/install/install_defender/install_cluster_container_defender.adoc

```diff
@@ -28,24 +28,30 @@ If you must manually specify a name, deploy your Defenders from *Manage > Defend
 . Go to *Manage > Defenders > Manage*.
 
 . Click *DaemonSets*.
++
+[IMPORTANT]
+====
+If you are using ARM architecture or multiple architectures you must edit the `daemonset.yaml` configuration file to https://cloud.google.com/kubernetes-engine/docs/how-to/prepare-arm-workloads-for-deployment#node-affinity-multi-arch-arm[prepare your workloads].
+====
 
 . For each cluster in the table, click *Actions > Deploy*.
 +
 The table shows a count of deployed Defenders and their version number.
 
-
 [.task]
+[#deamonset-gke]
 === Deploy Defender DaemonSet for GKE
 
 *Prerequisites:*
 
-* You have a GKE cluster deployed
-* You've created a corresponding Service Account key in JSON format. The Service Account should have the following permissions:
+* You deployed a GKE cluster
+* You prepared your workloads tu use https://cloud.google.com/kubernetes-engine/docs/how-to/prepare-arm-workloads-for-deployment#node-affinity-multi-arch-arm[ARM architecture or multiple architectures in GKE].
```

Comment:
@rcaballeromx I still don't think this is clear from this description. 
"You prepared your workloads" - it's not preparing the workloads, it's editing the defender.yeml daemonset. 
Maybe it will be more clear if we will add a section for GKE ARM and explain to the customers that they need to generate a yaml from the API / UI / twistcli according to the instructions specified in [Not sure where :( ] and add the instructions according to GKE

Reply:
It will be better to add this explanation to https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/install/install_kubernetes - Install the Prisma Cloud Defender.
the https://docs.paloaltonetworks.com/prisma/prisma-cloud/prisma-cloud-admin-compute/install/install_defender/install_cluster_container_defender document is just for automatic installation and not the regular installation flow. We will have to change this document to be more specific about that

Reply:
@MorShabiPAN I've updated the content as we discussed. As you can see, I needed to make additional changes in the included file instead of making them on this topic.

Reply:
@rcaballeromx I still can't see the #498 changes in this PR

Reply:
@MorShabiPAN Thanks for letting me know that you also made changes to the cluster installation flow, I've updated the PR accordingly.